core/src/main/java/com/confighub/core/repository/Property.java

```diff
@@ -296,7 +296,7 @@ public int hashCode()
     @Override
     public boolean equals( Object o )
     {
-        if ( null == o || !( o instanceof Property ) )
```

Comment:
why not a possibility of the hash code being null? now that things aren't not lazily loaded?

Reply:
hashCode() only deals with "this".  it can't be null if it's being called.

in equals(), "instanceof" already checks null.